app/controllers/ProofOfAuthorityController.scala

```diff
@@ -65,17 +65,17 @@ class ProofOfAuthorityController @Inject() (
   // GET /upload-proof-of-authority/file-verification
   final def showWaitingForFileVerification() = (identify andThen getData andThen requireData).async {
     implicit request =>
-      implicit val timeout = Timeout(10 seconds)
+      implicit val timeout = Timeout(appConfig.fileUploadTimeoutSeconds seconds)
```

Comment:
OBSERVATION: See above comment.  This is where we get from config in millisecond, convert it into seconds then use it.  Can this not be Timeout(appConfig.fileUploadTimeoutMilliseconds milliseconds) and change the AppConfig to read the milliseconds value?